packages/core/src/Chip/index.tsx

```diff
@@ -46,17 +47,30 @@ interface ChipProps extends Omit<BaseChipProps, 'component' | 'noPadding'> {
 
 /* eslint-disable-next-line react/display-name */
 export const Chip = React.forwardRef<HTMLDivElement, ChipProps>(
-  ({ text, error = false, onRemove, icon, ...props }, ref) => {
+  (
+    { text, error = false, disabled = false, onRemove, icon, ...props },
+    ref
+  ) => {
     const component = useMemo(() => {
       return (
         <>
           {!error && icon !== undefined ? <ChipIcon icon={icon} /> : null}
           <Typography variant="chip-tag-text">{text}</Typography>
-          {onRemove ? <ChipRemoveIcon onClick={onRemove} /> : null}
+          {onRemove ? (
+            <ChipRemoveIcon disabled={disabled} onClick={onRemove} />
```

Comment:
Is it not enough to not pass `onClick={onRemove}` part when it's disabled?

Reply:
And get rid of the icon completely? Practically speaking, I'd say it's enough :)

Reply:
According to Martin Gustafsson, it's okey to get rid of  **X**  `icon` when the `chip` is _disabled_.

I'll close the PR